codespell_lib/data/dictionary.txt

```diff
@@ -6986,6 +6986,7 @@ informations->information
 informaton->information
 informtion->information
 inforrmation->information
+infos->info
```

Comment:
`info` itself is a bit informal, and I wouldn't be surprised if `infos` were a commonly used (informal) pluralization of the word

Reply:
for the record: we do have `informations->informations`. Though I do agree with you, I'm not hardlined about it either way.

Reply:
reverted in 36a825f